File: src/eu/mcft/sumoremote/AboutActivity.java
package eu.mcft.sumoremote;

import eu.mcft.sumoremote.R;
import android.app.Activity;
import android.content.SharedPreferences;
import android.content.pm.PackageManager.NameNotFoundException;
import android.os.Bundle;
import android.preference.PreferenceManager;
import android.widget.TextView;

public class AboutActivity extends PrefsAdjustedActivity
{
	private TextView versionNumber;
	
	@Override
	protected void onCreate(Bundle savedInstanceState)
	{
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_about);
		
		versionNumber = (TextView)findViewById(R.id.versionNumber);
		
		// getting version number
		try
		{
			versionNumber.setText("v" + this.getPackageManager()
				    .getPackageInfo(this.getPackageName(), 0).versionName);
		}
		catch (NameNotFoundException e)
		{
			e.printStackTrace();
		}
	}
}

File: app/src/main/java/eu/mcft/sumoremote/AboutActivity.kt
package eu.mcft.sumoremote

import eu.mcft.sumoremote.preferences.PrefsAdjustedActivity
import android.content.pm.PackageManager.NameNotFoundException
import android.os.Bundle
import android.view.View
import android.widget.TextView

class AboutActivity : PrefsAdjustedActivity() {
    private var versionNumber: TextView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)

        versionNumber = findViewById<View>(R.id.versionNumber) as TextView

        // getting version number
        try {
            versionNumber!!.text = "v" + this.packageManager
                    .getPackageInfo(this.packageName, 0).versionName
        } catch (e: NameNotFoundException) {
            e.printStackTrace()
        }

    }
}
